out/zdt/push.go

```diff
@@ -0,0 +1,55 @@
+package zdt
+
+import (
+	"fmt"
+	"os/exec"
+)
+
+func CanPush(
+	cf func(args ...string) *exec.Cmd,
+	currentAppName string,
+) bool {
+
+	if currentAppName == "" {
+		return false
+	}
+
+	findErr := cf("app", currentAppName).Run()
+	appExists := findErr == nil
+
+	return appExists
+}
+
+func Push(
+	cf func(args ...string) *exec.Cmd,
+	currentAppName string,
+	pushFunction func() error,
+	showLogs bool,
+) error {
+
+	venerableAppName := fmt.Sprintf("%s-venerable", currentAppName)
+
+	actions := []Action{
```

Comment:
This seems to be based on the [actions from `autopilot`](https://github.com/contraband/autopilot/blob/1dc8b7d7d1636dde8b3d53e7a85cd62e7d5d647d/autopilot.go#L38) but it only implements some of it - is that intentional? Was the extra logic not necessary?